From bf0c87d38c592e307418ec7d6ec55a9b7f6309f0 Mon Sep 17 00:00:00 2001
From: Antoine Jacoutot <ajacoutot@gnome.org>
Date: Fri, 4 May 2012 11:10:48 +0200
Subject: [PATCH] evolution plugin: include missing header

To prevent liborg-freedesktop-Tracker-evolution-plugin.so:
undefined symbol 'E_MAIL_BACKEND'
liborg-freedesktop-Tracker-evolution-plugin.so':
Cannot load specified object

https://bugzilla.gnome.org/show_bug.cgi?id=675437
---
 src/plugins/evolution/tracker-evolution-plugin.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/src/plugins/evolution/tracker-evolution-plugin.c b/src/plugins/evolution/tracker-evolution-plugin.c
index 9e31753..db93c05 100644
--- a/src/plugins/evolution/tracker-evolution-plugin.c
+++ b/src/plugins/evolution/tracker-evolution-plugin.c
@@ -70,6 +70,7 @@
 #endif
 
 /* Remaining includes which are in ALL versions: */
+#include <mail/e-mail-backend.h>
 #include <shell/e-shell.h>
 
 #include <e-util/e-config.h>
-- 
1.7.10

